### PR TITLE
tiny update

### PR DESCRIPTION
**Pull request by siddontang**
@ngaut  @disksing

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,7 +1,7 @@
 all: build check test 
 
 build:
-	go build ./...
+	go build -o bin/pd-server pd-server/main.go
```
Comment (disksing) [inline Makefile]: Build client too?

Comment (siddontang) [inline Makefile]: client is a lib, not bin.

Comment (disksing) [inline Makefile]: Right..

**Comment by disksing** [comment]
LGTM

**Comment by ngaut** [comment]
LGTM

**Pull request merged**